README.md

```diff
@@ -390,8 +384,27 @@ watchdog.checkIsAnyThreadIsBlocked(); // check if any thread has exceeded the ti
 watchdog.tick(this); // In your custom thread class make incrementation to inform that the thread is not blocked
 ```
 
+# Working with NDK applications <a name="working_with_ndk"></a>
+
+## Enabling native integration
+
+If you would like to capture NDK Crashes you can use the `BacktraceDatabase` `setupNativeIntegration` method.
+
+```java
+database.setupNativeIntegration(backtraceClient, credentials);
+```
+
+In addition, you may need to add the [extractNativeLibs](https://developer.android.com/guide/topics/manifest/application-element#extractNativeLibs) option to your AndroidManifest.xml:
+```xml
+<application
+        android:extractNativeLibs="true">
+        ...
+</application>
+```
+More details about [extractNativeLibs](https://developer.android.com/guide/topics/manifest/application-element#extractNativeLibs) are available from the Android documentation
+
 ## Uploading symbols to Backtrace
-If you're developing a NDK application, to have better debugging experience in Backtrace, we recommend to upload application symbols to Backtrace. You can find application symbols in the "%application_dir%/build/intermediates/symbols/%release_type%/". To learn more about symbolification please check the article: https://help.backtrace.io/product-guide/symbolification
+For an NDK application, debugging symbols are not available to Backtrace by default. You will need to upload the application symbols for your native code to Backtrace. You can do this by uploading the native libraries themselves, which are usually found in the .apk bundle. [Click here to learn more about symbolification](https://help.backtrace.io/product-guide/symbolification)
 
```

Comment:
Help documentation has moved from help.backareace.io to support.backtrace.io - Can you take a quick pass and update any other links as well? This one will go to https://support.backtrace.io/hc/en-us/articles/360040517071-Symbolication-Overview